app/database/operator/app_data_operator/index.ts

```diff
@@ -14,10 +13,12 @@ const {APP: {INFO, GLOBAL}} = MM_TABLES;
 
 export default class AppDataOperator extends BaseDataOperator {
     handleInfo = ({info, prepareRecordsOnly = true}: HandleInfoArgs) => {
-        if (!info.length) {
-            throw new DataOperatorException(
-                'An empty "values" array has been passed to the handleInfo',
+        if (!info?.length) {
+            // eslint-disable-next-line no-console
+            console.warn(
+                'An empty or undefined "info" array has been passed to the handleInfo',
             );
+            return Promise.resolve([]);
```

Comment:
0/5 this could also have been resolved either specifying the function as async, or specifying the return type.

Reply:
Actually I'm going to change this to be async